src/test/java/org/jitsi/meet/test/mobile/base/AbstractBaseTest.java

```diff
@@ -41,15 +43,63 @@
     private static Logger logger
         = Logger.getLogger(AbstractBaseTest.class.getName());
 
+    /**
+     * The current room name used.
+     */
+    private static String currentRoomName;
+
     /**
      * Current mobile driver instance.
      */
     private static AppiumDriver<WebElement> driver;
 
     /**
-     * The current room name used.
+     * The OS currently being tested ('android' or 'ios').
      */
-    public static String currentRoomName;
+    private static String os;
```

Comment:
platform? since 'web' can also be a valid one, I guess?

Reply:
I next PR this will go away, because it will be possible to run Android, iOS and web all at the same time. There will be a separate class called MobileParticipant that will be dealing with that.

Reply:
Ack.